app/views/users/_roles.html.slim

```diff
@@ -7,5 +7,9 @@
       label.radio
         = radio_button_tag :role, role == 'all' ? '' : role, params[:role] == role
         = role.capitalize
-
+    h4 Coach availability
+    label.radio
+      = radio_button_tag :availability, true
```

Comment:
I'm not sure about the radio button. 1st, you cannot uncheck it. 2nd, I would imagine that a call-to-action button is better: a button "I am available for the upcoming season" with a direct `PUT` action being triggered (ideally via AJAX so that we don't get a page reload)

Reply:
I think you are talking about the checkbox btn that refers to coach availability update. 
This view that you commented above belongs to the filter page.
I removed the checkbox for availability and did an ajax call. I am not sure if that is what you mean...

![availability_btn](https://user-images.githubusercontent.com/15239141/30298977-f0536e5e-9723-11e7-9d90-014a1998cd35.png)


![availability_label](https://user-images.githubusercontent.com/15239141/30299006-18394cb8-9724-11e7-83c4-36625067e9c7.png)

Reply:
Yes, this is exactly what I meant <3